onions.md

```diff
@@ -11,7 +11,7 @@ To achieve this, ONIONS will serve as the focal point for network and service ab
 1. providing a comprehensive overview of operational needs and motivations for network and service abstractions,
 1. structuring efforts on service and network models, including their maintenance and how they interact with underlying device models,
 1. defining and maintaining models for network and service abstractions (i.e., at least the following models will be maintained by the WG: L2VPN Service Model (L2SM), L3VPN Service Model (L3SM), L2VPN Network Model (L2NM), L3VPN Network Model (L3NM), Attachment Circuit-as-a-Service (ACaaS), Service Attachment Points (SAP), Network Slicing, Service Assurance for Intent-Based Networking (SAIN), Virtual Network (VN), Abstraction and Control of TE Networks (ACTN), and the automating service and network management framework), and
-1. exploring approaches for mapping between various abstraction layers to enable seamless interoperability.
+1. specifying approaches for mapping between various abstraction layers to enable seamless interoperability.
```

Comment:
Ok, 2 questions: 
(1) Should it plural - "approaches", especially if we use specify! 
(2) What are your thoughts on s/interoperability/translation/ ; This comment from Qin makes sense to me.

Reply:
But Qin still prefers "“enable seamless interoperability and integration“

Reply:
* "approaches" gives us more flexibility and leaves to the decide about one or more.
* I agree "translation" is more concrete. I was initially OK with interoperability because I know that there are approaches that do not include translation (eg., use Device models as network models in which mount is used). Maybe that's also a form of translation.

Reply:
> and integration

mention integration makes sense to me.

Reply:
Made this change in the commit below.